addons/l10n_cl/data/template/account.tax-cl.csv

```diff
@@ -1,101 +1,101 @@
+"","","","","","","","","","","","tax","invoice","+Retenciones ILA (ventas)","account_210760","","","",""
+"","","","","","","","","","","","base","refund","-Base Retenciones ILA (ventas)","","","","",""
+"","","","","","","","","","","","tax","refund","-Retenciones ILA (ventas)","account_210760","","","",""
+"ila_v_205_s","20.5% ILA","Wines (Sales)","ILA S 20.5%","20.5","percent","sale","25","tax_group_ila","","7","base","invoice","+Base Retenciones ILA (ventas)","","","Vinos (Ventas)","20.5% ILA","ILA V 20.5%"
+"","","","","","","","","","","","tax","invoice","+Retenciones ILA (ventas)","account_210760","","","",""
+"","","","","","","","","","","","base","refund","-Base Retenciones ILA (ventas)","","","","",""
+"","","","","","","","","","","","tax","refund","-Retenciones ILA (ventas)","account_210760","","","",""
+"ila_l_315_s","31.5% ILA","Liquors 31.5% (Sales)","ILA S 31.5%","31.5","percent","sale","24","tax_group_ila","","7","base","invoice","+Base Retenciones ILA (ventas)","","","Licores 31.5% (Ventas)","31.5% ILA","ILA V 31.5%"
+"","","","","","","","","","","","tax","invoice","+Retenciones ILA (ventas)","account_210760","","","",""
+"","","","","","","","","","","","base","refund","-Base Retenciones ILA (ventas)","","","","",""
+"","","","","","","","","","","","tax","refund","-Retenciones ILA (ventas)","account_210760","","","",""
+"iva_compra_no_recup","19% NR","VAT Purchase 19% Non-recoverable","VAT 19% non-recoverable purchase","19.0","percent","purchase","14","tax_group_iva_19","","6","base","invoice","+Compras IVA No Recuperable","","","IVA Compra 19% No Recup.","19% IVA NR","IVA 19% NoR"
+"","","","","","","","","","","","tax","invoice","+IVA Pagado No Recuperable","account_420220","","","",""
+"","","","","","","","","","","","base","refund","-Compras IVA No Recuperable","","","","",""
+"","","","","","","","","","","","tax","refund","-IVA Pagado No Recuperable","account_420220","","","",""
+"iva_compra_uso_comun","19% C","VAT Purchase 19% Common Use","VAT 19% purchase for public use","19.0","percent","purchase","14","tax_group_iva_19","","6","base","invoice","+Compra Netas Gravadas Con IVA Uso Comun","","","IVA Compra 19% Uso Común","19% IVA UC","IVA 19% CUC"
+"","","","","","","","","","","","tax","invoice","+IVA Pagado Compras Uso Común","account_110730","","","",""
+"","","","","","","","","","","","base","refund","-Compra Netas Gravadas Con IVA Uso Comun","","","","",""
+"","","","","","","","","","","","tax","refund","-IVA Pagado Compras Uso Común","account_110730","","","",""
+"iva_supermercado_recup","19% SM","VAT Purchase 19% Supermarket","VAT 19% supermarket recup","19.0","percent","purchase","14","tax_group_iva_19","","6","base","invoice","+Compras De Supermercado","","","IVA Compra 19% Supermarket","19% IVA SupMRec","IVA 19% SupMRec"
```

Comment:
Hey @aliyatastemirova 

Thanks for the PR, on my side (and this was my mistake, sorry!) we should only translate the part that says "IVA Compra 19% Supermarket" to "IVA Compra 19% Supermercado Recup".

Thanks a lot!

Fernando